src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final Figure DEFAULT_FIGURE = new Circle(10, Colors.WHITE);
```

Comment:
Figure or Circle can not be constant - we can change inner state of this object. It means this object is mutable.
What we need is to create DEFAULT_RADIUS and DEFAULT_COLOR constants